Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -39,185 +38,161 @@ public void Generate_NoAnalyzersFoundInPackage_GenerateFails()
            
             NuGetPackageHandler nuGetHandler = new NuGetPackageHandler(remoteRepoBuilder.FakeRemoteRepo, GetLocalNuGetDownloadDir(), logger);
             AnalyzerPluginGenerator apg = new AnalyzerPluginGenerator(nuGetHandler, logger);
+            ProcessedArgs args = CreateArgs("no.analyzers.id", "0.9", "cs", null, false, outputDir);
 
             // Act
-            bool result = apg.Generate(new NuGetReference("no.analyzers.id", new SemanticVersion("0.9")), "cs", null, outputDir);
+            bool result = apg.Generate(args);
 
             // Assert
             Assert.IsFalse(result, "Expecting generation to fail");
             logger.AssertWarningsLogged(1);
             AssertSqaleTemplateDoesNotExist(outputDir);
-
         }
 
         [TestMethod]
-        public void Generate_PackageNoAccept_NoDependencies_Succeeds()
+        public void Generate_LicenseAcceptanceNotRequired_Succeeds()
```

Comment:
New tests added to cover license acceptance. I deleted all of the old tests, so a side-by-side comparison of the rest of this file won't make sense.